src/test/java/org/example/service/UserServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package org.example.service;
+
+import mock.UserRepositoryMock;
+import org.example.model.User;
+import org.example.repository.dao.UserRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceTest {
+    private List<User> users;
+    private UserService usersService;
+    @BeforeEach
+    void setUp() {
+        users = List.of(User.builder().nickname("dasha1").password("qwerty").build());
+        usersService = new UserService(new UserRepositoryMock(users));
+    }
+
+    @AfterEach
+    void tearDown() {
+       usersService = null;
+    }
+
+    @Test
+    void findUserIsPresent() {
```

Comment:
All names of the tests should have names like **test**FindUserIsPresent

Reply:
I'm not sure, but I thought it was enough that there is an annotation @Test above the method

Reply:
For this test method prefix test would be nice, in my opinion.
At the work we use naming convention like that: here is a good [article](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea), about unit test naming convention